spec/factories/purpose_config_factories.rb

```diff
@@ -340,6 +340,13 @@
         name { 'Pool tube' }
         creator_class { 'LabwareCreators::PooledTubesFromWholeTubes' }
       end
+
+      # Configuration for an aggregation tube (this is _not_ a thing)
+      factory :aggregation_tube_purpose_config do
```

Comment:
what's this factory used for again? doesn't seem to be mentioned elsewhere in this PR.
plus as we discussed before it's going to confuse anyone who searches for 'Aggregation' as it is definitely not a thing that is done with tubes